src/main/scala/loamstream/model/execute/ExecutionCell.scala

```diff
@@ -0,0 +1,52 @@
+package loamstream.model.execute
+
+import loamstream.model.jobs.LJob
+import loamstream.model.jobs.JobStatus
+import loamstream.model.jobs.JobResult
+
+/**
+ * @author clint
+ * Jan 24, 2020
+ * 
+ * A snapshot of an LJob's state at a particular moment during execution, from the perspective of RxExecuter and
+ * ExecutionState.  Tracks the jobs status and the number of times it has ran. 
+ */
+final case class ExecutionCell(status: JobStatus = JobStatus.NotStarted, runCount: Int = 0) {
```

Comment:
Just a naming question:

This is `ExecutionCell`, but all the data in it appears to be "state" and flags. There is also an `ExecutionState` class, which has a function that returns a list of `ExecutionCell` objects (one per job?). 

Is the relationship here more akin to `JobExecutionState` (Cell) and `JobListExeuctionState` (State)?

Reply:
Note: I'm not suggesting a rename, just trying to understand the relationship.

Reply:
I wanted `ExecutionState` to represent the state of a whole execution.  I think that name makes sense, or at least, I don't want its name to expose the details of some particular implementation.

`ExecutionCell` I'm less fond of, and `JobExecutionState` is probably better.  The reason I didn't do this at first is that there are already a few classes like `JobStatus`, and I didn't want to add another `Job*` thing with a name was very similar to others'.

Still, there's `ExecutionState.JobState`; I'll see if I can `ExecutionCell` into that, so there's just `JobState`.

Reply:
`ExecutionState.JobState` + `ExecutionCell` => `JobExecutionState`